src/openapi-generator.ts

```diff
@@ -853,6 +854,35 @@ export class OpenAPIGenerator {
       };
     }
 
+    if (isZodType(zodSchema, 'ZodTuple')) {
+      const { items } = zodSchema._def;
+
+      const tupleLength = items.length;
+
+      const schemas = items.map(schema => this.generateInnerSchema(schema));
+
+      const uniqueSchemas = uniq(schemas);
+
+      if (uniqueSchemas.length === 1) {
+        return {
+          type: 'array',
+          items: uniqueSchemas[0],
+          minItems: tupleLength,
+          maxItems: tupleLength,
+        };
+      }
+
+      return {
+        ...this.mapNullableType('array', isNullable),
+        // TODO: Compact is extra
```

Comment:
Sorry, what?

Reply:
There was a `compact` which was extra :smile: Now there is no compact ... but there was not TODO for the comment being extra. I'll remove it, thanks